src/styles/index.css

```
@@ -105,6 +106,8 @@
   --color-heading-black: black;
   --color-accent: #d1dce5;
   --color-accent-soft: #e7ecef;
+  --color-t28-dev-bg: rgb(249, 249, 251);
```

Stylelint (17.2.0)

[error] Expected modern color-function notation (color-function-notation)

(color-function-notation)